lib/update/clusterconfig/phases/update.go

```diff
@@ -0,0 +1,142 @@
+/*
+Copyright 2019 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package phases
+
+import (
+	"context"
+	"io"
+
+	"github.com/gravitational/gravity/lib/app"
+	libfsm "github.com/gravitational/gravity/lib/fsm"
+	"github.com/gravitational/gravity/lib/loc"
+	"github.com/gravitational/gravity/lib/ops"
+	"github.com/gravitational/gravity/lib/pack"
+	"github.com/gravitational/gravity/lib/schema"
+	"github.com/gravitational/gravity/lib/storage"
+
+	"github.com/gravitational/trace"
+	log "github.com/sirupsen/logrus"
+)
+
+// NewUpdateConfig returns a new executor to update runtime configuration on the specified node
+func NewUpdateConfig(
```

Comment:
I chose to still duplicate the configuration update phase for `ClusterConfiguration` as it does a couple of things differently and it was not worth creating another abstraction.
In the longer run, this will also enable to integrate the cluster update operation with the generic rollingupdate package.